pkg/cmd/tnctl/verify/revision.go

```diff
@@ -629,7 +632,7 @@ func (o *RevisionCommand) checkSecurityPolicy(ctx context.Context) error {
 			failed := gjson.GetBytes(results, "results.failed_checks")
 			if failed.Exists() && failed.IsArray() {
 				if len(failed.Array()) > 0 {
-					v.Info("Check ID against documentation: https://docs.bridgecrew.io/docs")
+					v.Info("Check ID against documentation at https://www.checkov.io")
```

Comment:
Would it be better to direct link to the list? https://www.checkov.io/5.Policy%20Index/all.html